docs/concepts/events-and-resources.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Events are the primary units of observability in Prefect Cloud
+tags:
+    - resources
+    - events
+    - observability
+---
+
+# Events
+
+Events are the primary unit of observability in Prefect Cloud, and are designed to be a simple but complete record of something occuring somewhere in your stack at a given time, and power several features in prefect cloud, including flow run logs, audit logs, and automations. Prefect events can represent API calls, state changes, or events from your execution environment or infrastructure. Events enable analysis of your data stack via [event feed](/ui/events/), and the confiuration of stack reactivity via [automations](/ui/automations/).
```

Comment:
- capitalization on `prefect cloud`
- "via the event feed" (added "the")
- spelling of `configuration`